immichpy/cli/commands/albums.py

```
@@ -249,7 +270,7 @@ def remove_asset_from_album(
 @app.command("remove-user-from-album", deprecated=False, rich_help_panel="API commands")
 def remove_user_from_album(
     ctx: typer.Context,
-    id: str = typer.Argument(..., help=""""""),
+    id: UUID = typer.Argument(..., help=""""""),
```

Ruff (0.15.20)

[error] Function argument `id` is shadowing a Python builtin

(A002)